knowage-vue/src/modules/documentExecution/dashboard/widget/SelectorWidget/SelectorWidget.vue

```diff
@@ -0,0 +1,451 @@
+            if (this.editorMode) return
+            updateStoreSelections(this.createNewSelection([this.selectedValue]), this.activeSelections, this.dashboardId, this.setSelections, this.$http)
+        },
+        multiValueSelectionChanged() {
+            if (this.editorMode) return
+            const tempSelection = this.createNewSelection(this.selectedValues)
+
+            this.updateActiveSelectionsWithMultivalueSelection(tempSelection)
+        },
+        dateSelectionChanged() {
+            if (this.editorMode) return
+            updateStoreSelections(this.createNewSelection([moment(deepcopy(this.selectedDate)).format(dashboardDescriptor.selectionsDateFormat)]), this.activeSelections, this.dashboardId, this.setSelections, this.$http)
+        },
+        dateRangeSelectionChanged() {
+            if (this.editorMode) return
+            const tempDateValues = [] as string[]
+            for (let i = 0; i < this.initialOptions.rows.length; i++) {
+                const iniitalOption = this.initialOptions.rows[i].column_1
+                const tempDate = moment(iniitalOption, dashboardDescriptor.selectionsDateMultiFormat).valueOf()
+                if ((!this.startDate || tempDate >= this.startDate.getTime()) && (!this.endDate || tempDate <= this.endDate.getTime())) tempDateValues.push(iniitalOption)
+            }
+            const tempSelection = this.createNewSelection(tempDateValues)
+            updateStoreSelections(tempSelection, this.activeSelections, this.dashboardId, this.setSelections, this.$http)
+        },
+        updateActiveSelectionsWithMultivalueSelection(tempSelection: ISelection) {
+            const index = this.activeSelections.findIndex((activeSelection: ISelection) => activeSelection.datasetId === tempSelection.datasetId && activeSelection.columnName === tempSelection.columnName)
+            if (index !== -1) {
+                this.activeSelections[index] = tempSelection
+            } else {
+                this.activeSelections.push(tempSelection)
+            }
+        },
+        createNewSelection(value: (string | number)[]) {
+            return { datasetId: this.propWidget.dataset as number, datasetLabel: this.getDatasetLabel(this.propWidget.dataset as number), columnName: this.propWidget.columns[0]?.columnName ?? '', value: value, aggregated: false, timestamp: new Date().getTime() }
+        },
+        getDatasetLabel(datasetId: number) {
+            const index = this.datasets.findIndex((dataset: IDataset) => dataset.id.dsId == datasetId)
+            return index !== -1 ? this.datasets[index].label : ''
+        },
+        onSelectionsDeleted(selections: any) {
+            const index = selections.findIndex((selection: ISelection) => selection.datasetId === this.propWidget.dataset && selection.columnName === this.propWidget.columns[0]?.columnName)
+            if (index !== -1) this.removeDeafultValues()
+        }
+    }
+})
+</script>
+
+<style lang="scss">
+.selector-widget {
```

Comment:
Is it possible to put the selector-widget class into a specific __selector-widget.scss_ file?

Reply:
Fixed, I named it _selectorWidget.scss to follow the naming convention (some other file also used camel case)